core/src/main/java/io/aiven/kafka/tieredstorage/fetch/FetchManager.java

```diff
@@ -23,9 +23,9 @@
 import io.aiven.kafka.tieredstorage.storage.ObjectKey;
 import io.aiven.kafka.tieredstorage.storage.StorageBackendException;
 
-public interface ChunkManager {
-
-    InputStream getChunk(final ObjectKey objectKey,
-                         final SegmentManifest manifest,
-                         final int chunkId) throws StorageBackendException, IOException;
+public interface FetchManager {
+    InputStream partContent(final ObjectKey objectKey,
```

Comment:
Maybe the method should be called `fetchPartContent`?

Reply:
Sure, good idea.